agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml

```diff
@@ -65,13 +65,14 @@ spec:
         securityContext:
           privileged: true
         resources:
-          # Resources needed are subjective on the actual workload
-          # please refer to Sysdig Support for more info about it
+          # Resources needed are subjective on the actual workload,
+          # please contact Sysdig Support for tuning these numbers.
```

Comment:
These are two seperate statements; you can't use a comma.  Also "subjective on" and "for tuning"  feel like odd phrases.

Reply:
At the end of the day, these are customer facing. Maybe AmyL should tweak it for us.